tasks/main.yml

```diff
@@ -143,3 +143,30 @@
     name: "{{ fluentd_service_name }}"
     enabled: true
     state: started
+
+- name: Check if pid directory exists
+  stat:
+    path: "{{ fluentd_pid_dir }}"
+  register: register_fluentd_stat_pid_dir
+  changed_when: false
+  when: not ansible_check_mode
+
+- name: Assert pid directory exists
+  assert:
+    that:
+      - register_fluentd_stat_pid_dir.stat.exists
+      - register_fluentd_stat_pid_dir.stat.isdir
+  when: not ansible_check_mode
+
+- name: Check if pid file exists
```

Comment:
as commented in cb74d94cd09b1a671f1b7780fd7f4036b423927d, they are a test in `ansible` play. `serverspec` helps role development a lot but does not when used in a project. we should have [more tests in roles](http://docs.ansible.com/ansible/test_strategies.html) as well.